packages/core/src/chains/cosmos/sdkCosmos/query/IQuery.ts

```
@@ -52,11 +52,12 @@ class QueryRegistry {
 export const queryRegistry = new QueryRegistry();
 const cosmwasmQuery = new CosmwasmQuery();
 queryRegistry.register(getNetworkIdsMap().juno, cosmwasmQuery);
-// queryRegistry.register(getNetworkIdsMap().terra, cosmwasmQuery); // terra2
+//queryRegistry.register(getNetworkIdsMap().terra, cosmwasmQuery);  // Terra Classic
```

ESLint

[error] Expected exception block, space or tab after '//' in comment.

(spaced-comment)